### DISCUSSION TITLE
Enable 2fa by Text messages (SMS) for Türkiye

### DISCUSSION THREAD

**Discussion by FaithSoftwareCo** [community/community#86778] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello, GitHub doesn't support 2fa enable by sms verification for people in Turkey. Please add this support...

**Comment by mayankzarkar** [comment]
Hello @FaithSoftwareCo 

GitHub's 2FA supports various methods, but it may not specifically support SMS-based 2FA in all regions.

If GitHub's 2FA with SMS verification is not available in Turkey, it could be due to various reasons, such as regulatory considerations, security concerns, or limitations imposed by the service provider.

**Here are some steps you can take:**

**Check GitHub Documentation:** Review GitHub's official documentation on two-factor authentication to see if there have been any updates or changes to the supported methods.

**Contact GitHub Support:** If you have specific concerns or requests regarding 2FA in your region, you can contact GitHub support. They will be able to provide information on the current status and any plans for future support in your region.

**Explore Alternative Authentication Methods:** In the meantime, consider using other 2FA methods supported by GitHub, such as TOTP (e.g., using authenticator apps like Google Authenticator or Authy) or security keys.

Thanks

**Comment by ghostinhershell** [comment]
Hi @FaithSoftwareCo 👋🏾 

Thank so much for writing in. If your country is not on [the list](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/countries-where-sms-authentication-is-supported#supported-countries-for-sms-authentication), then we aren't currently able to reliably deliver text messages to your country. We track delivery rates across all countries, including ones not in the list, and review it to understand when we have to remove countries or can add them back. We do not add countries with low deliverability rates, because it leads to users being locked out of their accounts when they can’t receive the SMS for 2FA.

If GitHub doesn’t support two-factor authentication via text message for your country of residence, you must set up authentication via a TOTP application. For more information on how to configure 2FA, please review our [documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-mobile-app).

You can also find this, and other frequently asked questions, in our [Two Factor Authentication Frequently Asked Questions 🔑 🗝️](https://github.com/orgs/community/discussions/68240) post.

Let me know if you have any additional questions. Thanks!
